tests/testthat/test-datatable-disconnection.R

```diff
@@ -0,0 +1,84 @@
+test_that("`coerce_dt` gives new `data.table` object", {
+  dummy <- data.table::data.table(dummy = 1:10)
+  newdt <- coerce_dt(dummy)
+  expect_false(data.table::address(newdt) == data.table::address(dummy))
+})
+
+test_that("`coerce_dt` gives new `data.table` object, unless asked not to", {
```

Comment:
everywhere else we haven't used '' to highlight functions in our test suite. Perhaps we should (world least fun new issue) but for style should follow everywhere else convention and drop here I think.

Reply:
alright, more grist for the style guide